docs/overview/quick_start/quick_start.mdx

```diff
@@ -30,7 +33,7 @@ There are a few things to notice here!
   * Most importantly, we've decorated this function with `@bm.random_variable`. This is how you tell Bean Machine to interpret this function probabilistically. `@bm.random_variable` functions are the building blocks of Bean Machine models, and let the framework explore different values that the function represents when fitting a good distribution for observed data that you'll provide later.
   * Next, notice that the function returns a [PyTorch distribution](https://pytorch.org/docs/stable/distributions.html?highlight=distribution#module-torch.distributions). This distribution encodes your prior belief about a particular random variable. In the case of $\text{Exponential}(10.0)$, our prior has this shape:
 
-<img src={useBaseUrl("/img/prior_exponential.png")} />
+<PriorExponentialPlot />
```

Comment:
I agree about your suggestion making the readability better. I will look into this.

Reply:
I'm not sure what the best pattern is in all honesty. I like the separation of React components in their own file and then the mdx file importing the components it needs. This is the pattern that the automated script does when converting `.ipynb` files into `.mdx` files. Based solely on trying to mimic that pattern, my preference is to keep the components in a separate file.

But I can be swayed either way if there is momentum to change the pattern.

Reply:
I see :). I still think it would be cleaner to reuse the same `BokehFigure` component, which is pretty much analogous to using the `img` DOM element. It wasn't too big of a deal when you did the `ipynb` to `mdx` conversion, because the files are being generated and are not checked into the repository. However, since we are writing these docs manually, it's probably going to be easier if our devs don't have to create a new component every time we want to add a plot.

Reply:
That is a solid reason for the change, and one I did not think about. I'll make the those changes as soon as I can.

Reply:
I got the `BokehFigure` component use as close as I could to `<img src=... />`. In order to get the docs to build, we have to use the component in the following way in the mdx files.

```js
<BokehFigure data={React.useMemo(() => require("./path/to/data"), [])} />
```

Let me know if this is okay.

Reply:
Removing the `React.useMemo()` worked just fine. The initial reason why we added it was due to a re-rendering of the figures if the page link was clicked or reloaded, that caused figures to be duplicated on the page. The memo on the BokehFigure I think is all that is needed to prevent duplicate images from appearing on the page if it has been reloaded.